README.md

```diff
@@ -407,6 +407,34 @@ module Maintenance
 end
 ```
 
+### Handling errors during iteration
+
+By default any errors encountered while processing an iteration will be raised and
+the task with error. Errors can be rescued using the `rescue_from` handler. Errors
+rescued this way will be reported by the handler and iteration will continue.
+
+The default handler just calls `Rails.error.report` with the raised error.
+
+```ruby
+module Maintenance
+  class UpdatePostRescueErrorTask < MaintenanceTasks::Task
+    rescue_from ActiveRecord::RecordNotUnique
+  end
+end
+```
+
+Or you can pass a custom handler with a block.
+
+```ruby
+module Maintenance
+  class UpdatePostRescueErrorTask < MaintenanceTasks::Task
+    rescue_from ActiveRecord::RecordNotUnique do |error|
+      CustomErrorHandler.report(error)
+    end
+  end
+end
+```
+
```

Comment:
Maybe we should add a note indicating that if the handler raises, the whole task _will_ error?